src/android/app/src/main/java/org/citra/citra_emu/overlay/InputOverlayDrawableButton.kt

```diff
@@ -30,6 +44,9 @@ class InputOverlayDrawableButton(
     val opacity: Int
 ) {
     var trackId: Int
+
+    private var isMotionFirstButton = false // mark buttons that did not get activated by sliding
```

Comment:
What is the purpose of this variable?
From the name it's not clear what exactly it's tracking.

Reply:
this is a flag for the third mode (KeepFirst: The first button is kept activated until released, further buttons use the simple button sliding method.) to track the first button that was activated to keep it activated even when sliding off until released.
The name was supposed to suggest that it was the first button that was activated with the current motion.

Reply:
comment was not updated properly when re-using and re-naming the variable

```suggestion
    private var isMotionFirstButton = false // mark the first activated button with the current motion
```